test/util/test_runners/test_runner_args.py

```diff
@@ -43,6 +44,8 @@ def create_from_args():
         parser = ArgumentParser(description='Test arguments')
 
         # Add the arguments
+        parser.add_argument('--oss', dest='run_oss_tests', action='store_true',
+                            help='Added to run OSS-specific tests')
```

Comment:
The jenkins build is still failing :( 
https://jenkins.conjur.net/blue/organizations/jenkins/cyberark--conjur-api-python3/detail/update_process_flow/2/pipeline

Reply:
now ok. assumed wrong about how nose running tests